CONTRIBUTING.md

```diff
@@ -110,31 +110,41 @@ All contributions to the What The Hack repo come through pull requests. This mea
+We have a GitHub Action that automates the creation of a new hack and "scaffolds" out all of the Markdown templates from the [WTH Author's Guide](https://aka.ms/wthauthor) for you. After that, you can open your [favorite text editor](https://code.visualstudio.com) and start plugging in your content.
+
+The instructions below assume you have the [Git command line tool](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Visual Studio Code](https://code.visualstudio.com/) installed on your machine. 
 
 1. Create a fork of the WTH repo
-   - Navigate to the WTH repo at: <https://aka.ms/wthrepo>
+   - Navigate to the WTH repo at: [https://aka.ms/wthrepo](https://aka.ms/wthrepo)
    - Click the "Fork" button at the top right of the page and then choose the account you want to create the fork in. 
-1. Clone your new fork to your local machine
+1. Navigate to your fork of the WTH repo in the browser at: 
+    ```https://github.com/<myGithubName>/WhatTheHack```
+1. Click on the "Actions" tab at the top of the page and then select "Create new hack" from the list of "All Workflows" that appears on the left side of the screen.
+1. Click on the "Run workflow" dropdown that appears on the right side of the page and fill in the form that appears with:
+    - The name of your hack. Do not use spaces in your hack's name.
+    - The number of challenges you would like your hack to have. We recommend selecting a higher number of challenges than you actually plan to have as it is easier to remove extra challenge templates then add new ones manually later.
+    
+    **NOTE:** Do NOT change the value of the "Run workflow from" field from the pre-selected "branch: master" 
+1. Click the green "Run Workflow" button on the form to kick off the GitHub Action. This process will take a couple of minutes to complete. The action will:
+    - Create a new branch in your fork with the name of your hack prefixed with "xxx-". For example: `xxx-MyAwesomeHack`
+    - Create a new folder in the new branch with the same name as the branch: `xxx-MyAwesomeHack`
+    - Create templated markdown files in the new folder for your hack as per the [WTH Author's Guide](https://aka.ms/wthauthor).
+1. When the GitHub Action has completed running, clone your new fork to your local machine. 
    ```
-   git clone https://github.com/<myname>/WhatTheHack.git
+   git clone https://github.com/<myGitHubName>/WhatTheHack.git
    cd WhatTheHack
    ```
-1. Create a new branch for your work in your fork. It is a best practice to never work directly on the main/master branch
+   
+   **NOTE:** Before running the `git clone` command, navigate to a folder location on your local workstation where you would like to clone it to.
+   
+   **WARNING:** Never clone a Git repo to a folder location that is synchronized by cloud service such as OneDrive, GoogleDrive, etc.
```

Comment:
Since the note and warning are quite significant.  I think we should move them above the "git clone" step or as part of the description.

Reply:
I agree! Almost did it myself earlier.  I'll make this change.

Reply:
Change made.